backend/src/main/resources/db/migration/V1__initializing_data.sql

```diff
@@ -203,6 +232,17 @@ comment on column nrfc.submission_detail.organization_name is 'The name of the c
 comment on column nrfc.submission_detail.client_type_code is 'A code representing the type of a client';
 comment on column nrfc.submission_detail.good_standing_ind is 'An indicator that determines whether a client is in good standing with respect to their financial obligations.';
 
+create table if not exists nrfc.submission_matching_detail (
+    submission_matching_detail_id        integer			not null,
+	submission_id				integer			not null,
+	matching_fields_json		text			null,
```

Comment:
Please rename it to be just `matching_fields` and change it's type to be `jsonb` instead of text

Reply:
Done!